examples/volume_RT_shim_respiratory.py

```diff
@@ -27,32 +13,32 @@
 # > phase = load_niftis( niftiPath ) 5D array (x, y, z, nEcho, nAcq)
 #
 # Convert to complex
-# complexArray = mag.*exp( 1i.*phase ); 5D array (x, y, z, nEcho, nAcq)
+# > complexArray = mag.*exp( 1i.*phase ); 5D array (x, y, z, nEcho, nAcq)
```

Comment:
what's "1i"?

Reply:
When creating complex on Matlab, it is better to use '1i' rather than 'i' to avoid confusion with eventual variables called 'i'. I don't know how we will do this in python though.

Reply:
In python is is j that is import from the cmath package. Also, in Python element wise operation doesn't exist. You'll have to use a separate function, a list comprehension or (easier) numpy.